salt/modules/test.py

```diff
@@ -469,18 +469,26 @@ def opts_pkg():
     return ret
 
 
-def rand_str(size=9999999999):
+def rand_str(size=9999999999, hash_type='md5'):
     '''
     Return a random string
 
+        size
+            size of the string to generate
+        hash_type
+            hash type to use
+
+            .. versionadded:: 2015.5.2
+
     CLI Example:
 
     .. code-block:: bash
 
         salt '*' test.rand_str
     '''
-    hasher = getattr(hashlib, __opts__.get('hash_type', 'md5'))
-    return hasher(str(random.SystemRandom().randint(0, size))).hexdigest()
+    return salt.utils.rand_str(
+        hash_type= __opts__.get('hash_type', hash_type),
```

Comment:
This looks to me like it will default to `hash_type` if it's present in opts, even if one is explicitely passed to this function. Is that what's intended here?